include/alp/structures.hpp

```diff
@@ -529,6 +529,24 @@ namespace alp {
 				>::type;
 		};
 
+		struct HermitianTridiagonal: BaseStructure {
+
+			private:
+
+				typedef Interval< -1, 2 > I;
+
+			public:
+
+				typedef std::tuple< I > band_intervals;
+
+				using inferred_structures = tuple_cat<
+					std::tuple< HermitianTridiagonal >,
+					Hermitian::inferred_structures,
+					Tridiagonal::inferred_structures,
+					Band< I >::inferred_structures
```

Comment:
Band is not needed as it is inferred from Tridiag (same holds for SymmetricTridiag above)

Reply:
Fixed, also for SymmTridiag